po/LINGUAS

```diff
@@ -22,3 +22,4 @@ uk
 zh_CN
 zh_TW
 
+cs
```

Comment:
That's weird. `cs` is already in the list.

Reply:
Fixed before push.